react/features/connection-indicator/actions.js

```diff
@@ -1,15 +1,19 @@
-import { CONNECTION_INDICATOR_SAVE_LOGS } from './actionTypes';
+import getRoomName from '../base/config/getRoomName';
+import { downloadJSON } from '../base/util/downloadJSON';
 
 
 /**
  * Create an action for saving the conference logs.
  *
- * @returns {{
- *     type: CONNECTION_INDICATOR_SAVE_LOGS
- * }}
+ * @returns {Function}
  */
 export function saveLogs() {
-    return {
-        type: CONNECTION_INDICATOR_SAVE_LOGS
+    return (dispatch, getState) => {
+        // this can be called from console and will not have reference to this
```

Comment:
Hmmm... Is this still the case? I don't believe in this version it can be called from the console.

Reply:
Oh maybe this is  left overs from the  conference.js. The method there can still be called  from the console. Maybe you should remove the comment.

Reply:
Oups, you're right, that comment sneaked in